dartagnan/src/main/java/com/dat3m/dartagnan/program/processing/Intrinsics.java

```diff
@@ -1293,12 +1294,17 @@ private List<Event> inlineNonDet(FunctionCall call) {
     //FIXME: The following support for memcpy, memcmp, and memset is unsound
     // For proper support, we need at least alias information and most likely also proper support for mixed-sized accesses
 
-    // Handles both std.memcpy and llvm.memcpy
-    private List<Event> inlineMemCpy(FunctionCall call) {
+    // Handles std.memcpy, std.memcpy_s and llvm.memcpy
+    private List<Event> memCpy(FunctionCall call, boolean c11) {
```

Comment:
How does `c11` play into this? From the documentation, it seems that `c99` does not even have the `_s` version of `memcpy`. Also, why did you get rid of the "inline"-suffix in the method name?

Reply:
Just to make it a bit more clear that this method was not directly called from the same place all other `inlineX` are called.
But I guess I can put back the inline in the name and use lambdas as you suggest in the other comment.